frontend/src/scenes/onboarding/Onboarding.tsx

```diff
@@ -78,6 +82,21 @@ const ProductAnalyticsOnboarding = (): JSX.Element => {
                 teamPropertyToVerify="ingested_event"
                 stepKey={OnboardingStepKey.VERIFY}
             />
+            <OnboardingProductConfiguration
+                stepKey={OnboardingStepKey.PRODUCT_CONFIGURATION}
+                options={[
+                    {
+                        title: 'Autocapture frontend interactions',
+                        description: `If you use our JavaScript or React Native libraries, we'll automagically 
+                            capture frontend interactions like pageviews, clicks, and more. Fine-tune what you 
+                            capture directly in your code snippet.`,
+                        teamProperty: 'autocapture_opt_out',
+                        value: !currentTeam?.autocapture_opt_out,
+                        type: 'toggle',
+                        inverseToggle: true,
+                    },
+                ]}
+            />
         </OnboardingWrapper>
     )
```

Comment:
Ha, github won't let me comment on the replay lines because they've not changed 🙈 

Would be cool to have the basic settings here 
<img width="437" alt="Screenshot 2023-11-30 at 10 08 05" src="https://github.com/PostHog/posthog/assets/984817/526ae6d2-e986-4df0-9022-2e6499d2d528">

Ultimately we'll want the cost controls here too

Reply:
Yep! Planning on picking this up again today 👍